web/src/app/converters/survey-data-converter.ts

```diff
@@ -53,31 +54,42 @@ export const MODEL_ROLES = Map([
   [Pb.Role.VIEWER, Role.VIEWER],
 ]);
 
-const MODEL_SHARING_MODEL_TYPES = Map([
+const MODEL_DATA_SHARING_TYPES = Map([
```

Comment:
This refers specifically to the terms, so should be included int he name - `MODEL_DATA_SHARING_TERMS_TYPES`